README.md

```diff
@@ -162,7 +162,7 @@ packages:
       version: [">=1.0.0", "<2.0.0"]
 ```
 
-# 🙌 How is this package maintained and can I contribute?
+# 🙌 How is this package maintained and can I contribute? 
```

Comment:
Would you be able to add a sub section in here to point users to the DECISIONLOG for opinionated decisions. 

See how we did this in [Zendesk](https://github.com/fivetran/dbt_zendesk#opinionated-modelling-decisions)